content/docs/introduction/comparison.md

```diff
@@ -132,14 +132,15 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
 options](https://github.com/influxdata/kapacitor/issues/277) for rules,
-alerting, or notifications. Prometheus and the Alertmanager by contrast offer a
-redundant option via running redundant replicas of Prometheus and using the
-Alertmanager's [High
-Availability](https://github.com/prometheus/alertmanager#high-availability)
-mode. In addition, Kapacitor can be scaled via manual sharding by the user,
-similar to Prometheus itself.
+alerting, or notifications.  [Enterprise Kapacitor](https://docs.influxdata.com/enterprise_kapacitor) 
+is Influx's commercial option for running a cluster of Kapacitors, for HA with
+alert deduplication.  Prometheus and the Alertmanager by contrast offer a 
+redundant option via running redundant replicas of Prometheus and using the 
+Alertmanager's [High Availability](https://github.com/prometheus/alertmanager#high-availability)
+mode. In addition, the open-source Kapacitor can be scaled via manual sharding 
```

Comment:
This is also the case for the commercial Kapacitor AFAICT. It is equivalent in this sense to Prometheus+AM.